src/test/java/com/zepben/evolve/services/common/translator/CimPbTest.kt

```diff
@@ -166,6 +166,7 @@ class CimPbTest {
                 is Jumper -> targetNetwork.addFromPb(it.toPb())
                 is Recloser -> targetNetwork.addFromPb(it.toPb())
                 is Breaker -> targetNetwork.addFromPb(it.toPb())
+                is LoadBreakSwitch -> targetNetwork.addFromPb(it.toPb())
```

Comment:
You have added the `when` condition, but you have not actually created a test to make use of it. I think this is a obsolete set of tests so probably not even worth doing (as in not worth adding it to the `when` like `BusbarSection` or `PowerElectronicsConnection` etc.)